### PR TITLE
Refine OpenGov doc

### PR DESCRIPTION
**Pull request by DrW3RK**
- [x] Remove ambiguity on Approval computation
- [x] Move conviction voting section above approval/support section (as calculation example requires understanding of it)
- [x] Add a simple example computing approval and support

### CODE REVIEW

File: docs/learn/learn-polkadot-opengov.md [hunk 1]
```diff
@@ -336,12 +412,19 @@ Decision Period.
...
-  [conviction](#voluntary-locking)) against the total vote-weight (_aye_, _nay_, and _abstained_).
+- **Approval** is defined as the share of [conviction](#voluntary-locking)-weighted _aye_ votes
+  against the conviction-weighted total of _aye_ and _nay_ votes. The code implementation can be viewed
+  [here](https://github.com/paritytech/polkadot-sdk/blob/f2fbba3be1d7deaf7cfc731cea00552c212ddfcf/substrate/frame/conviction-voting/src/types.rs#L77)
 - **Support** is the total number of _aye_ and _abstain_ votes (ignoring any adjustment for
   conviction) compared to the total possible votes ([active issuance](learn-DOT.md#token-issuance))
   that could be made in the system. In case of _split_ votes, only _aye_ and _abstain_ will count.
 
+For example, let us consider a hypothetical example where the total active issuance is {{ polkadot: 100 DOT :polkadot }}{{ kusama: 100 KSM :kusama }}
+and an account A with 10 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} votes "Aye" with 4x conviction, where as account B votes "Nay" with 5 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} with 2x conviction
```
Comment (filippoweb3) [inline docs/learn/learn-polkadot-opengov.md]:
```suggestion
and an account A with 10 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} votes "Aye" with 4x conviction, whereas account B votes "Nay" with 5 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} with 2x conviction 
```

File: docs/learn/learn-polkadot-opengov.md [hunk 2]
```diff
@@ -336,12 +412,19 @@ Decision Period.
...
+  against the conviction-weighted total of _aye_ and _nay_ votes. The code implementation can be viewed
+  [here](https://github.com/paritytech/polkadot-sdk/blob/f2fbba3be1d7deaf7cfc731cea00552c212ddfcf/substrate/frame/conviction-voting/src/types.rs#L77)
 - **Support** is the total number of _aye_ and _abstain_ votes (ignoring any adjustment for
   conviction) compared to the total possible votes ([active issuance](learn-DOT.md#token-issuance))
   that could be made in the system. In case of _split_ votes, only _aye_ and _abstain_ will count.
 
+For example, let us consider a hypothetical example where the total active issuance is {{ polkadot: 100 DOT :polkadot }}{{ kusama: 100 KSM :kusama }}
+and an account A with 10 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} votes "Aye" with 4x conviction, where as account B votes "Nay" with 5 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} with 2x conviction 
+and finally account C with 20 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} votes "abstain". In this scenario, only 45 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} from the total active issuance participated in voting on the
```
Comment (filippoweb3) [inline docs/learn/learn-polkadot-opengov.md]:
```suggestion
and finally account C with 20 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} votes "abstain". In this scenario, only 35 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} from the total active issuance participated in voting on the
```

File: docs/learn/learn-polkadot-opengov.md [hunk 3]
```diff
@@ -336,12 +412,19 @@ Decision Period.
...
+referendum. Now, let us calculate the Approval and Support values for that referendum. Approval is calculated as Aye'/ Aye' + Nay', where Aye' and Nay' are the votes after
```
Comment (filippoweb3) [inline docs/learn/learn-polkadot-opengov.md]:
```suggestion
referendum. Now, let us calculate the Approval and Support values for that referendum. Approval is calculated as (Aye') / (Aye' + Nay')  where "Aye'" and "Nay'" are the votes after 
```

File: docs/learn/learn-polkadot-opengov.md [hunk 4]
```diff
@@ -336,12 +412,19 @@ Decision Period.
...
   that could be made in the system. In case of _split_ votes, only _aye_ and _abstain_ will count.
 
+For example, let us consider a hypothetical example where the total active issuance is {{ polkadot: 100 DOT :polkadot }}{{ kusama: 100 KSM :kusama }}
+and an account A with 10 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} votes "Aye" with 4x conviction, where as account B votes "Nay" with 5 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} with 2x conviction 
+and finally account C with 20 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} votes "abstain". In this scenario, only 45 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} from the total active issuance participated in voting on the
+referendum. Now, let us calculate the Approval and Support values for that referendum. Approval is calculated as Aye'/ Aye' + Nay', where Aye' and Nay' are the votes after 
+the conviction multiplier is applied. Hence, Approval = (10 x 4) / (10 x 4 + 5 x 2) = 40/50 which is 80%. Support is calculated as (Aye + Abstain) / (total active issuance). Hence,  Support = (10 + 20) / 100 which is 30%.
```
Comment (filippoweb3) [inline docs/learn/learn-polkadot-opengov.md]:
```suggestion
the conviction multiplier is applied. Hence, Approval = (10 x 4) / (10 x 4 + 5 x 2) = 40/50 which is 80%. Support is calculated as (Aye + Abstain) / (total active issuance), where "Aye" and "Abstain" are the votes without the conviction multiplier. Hence,  Support = (10 + 20) / 100 which is 30%.
```

**Review by filippoweb3 — APPROVED** [review]
I added some suggestions. Additionally, I would present the example with bullet points. It eases the read.

File: docs/learn/learn-polkadot-opengov.md (hunk 3 above)
Comment (profradha) [reply docs/learn/learn-polkadot-opengov.md]: Aye' is Aye with conviction. It is important to differentiate this variable from just Aye which is in support formula, which corresponds to DOT used to vote Aye. I see you added in the description, but for someone who might only be looking at the formuals would not be aware that there is a difference between these variables.

File: docs/learn/learn-polkadot-opengov.md (hunk 1 above)
Comment (profradha) [reply docs/learn/learn-polkadot-opengov.md]: This is "whereas". I'll edit it.

**Pull request merged**